flow_apollo2_data_processing/processing/workflow.cwl

```diff
@@ -28,26 +28,28 @@ steps:
       [out_fai]  
   #step 3
   prepare-refseqs:
-    run: prepare-refseqs.cwl
+    run: prepare-refseqs.cwl               
     in:
       in_fasta: in_fasta
       in_fai: samtools_faidx/out_fai
     out: 
-      [out_trackList_json, out_seq, out_tracks_conf] 
-  #step 4, different gff ??????
+      [out_trackList_json, out_seq, out_tracks_conf,out_data]  
```

Comment:
Make data folder also become an output of prepare-refseqs.cwl.
Therefore, we can use this folder as an input in the next step flatfile-to-json.cwl.